src/components/connect.js

```diff
@@ -174,14 +174,34 @@ export default function connect(mapPropsToRequestsToProps, options = {}) {
               }, mapping.refreshInterval)
             }
 
-            this.setAtomicState(prop, startedAt, mapping, PromiseState.resolve(value, meta), refreshTimeout)
+            // TODO: re-think naming and params of then, catch, andThen, andCatch
```

Comment:
The naming of these makes sense to me, especially given the context that @ryanbrainard says inspired these functions:

http://www.scala-lang.org/files/archive/nightly/docs/library/index.html#scala.concurrent.Future